specification/maps/resource-manager/Microsoft.Maps/preview/2021-12-01-preview/maps-management.json

```diff
@@ -316,6 +316,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Maps/accounts/{accountName}/listSas": {
+      "post": {
+        "tags": [
+          "Accounts"
+        ],
+        "description": "Create and list an account shared access signature token. Use this SAS token for authentication to Azure Maps REST APIs through various Azure Maps SDKs. As prerequisite to create a SAS Token. \n\nPrerequisites:\n1. Create or have an existing User Assigned Managed Identity in the same Azure region as the account. \n2. Create or update an Azure Map account with the same Azure region as the User Assigned Managed Identity is placed.",
+        "operationId": "Accounts_ListSas",
+        "x-ms-examples": {
+          "List Account Sas": {
+            "$ref": "./examples/AccountListSAS.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "name": "MapsAccountSasParameters",
+            "in": "body",
+            "description": "The updated parameters for the Maps Account.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AccountSasParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful.",
+            "schema": {
+              "$ref": "#/definitions/MapsAccountSasToken"
```

Comment:
Yes expected, read more precedent examples here: https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#list

Specific example is AccountSas - https://docs.microsoft.com/en-us/rest/api/storagerp/storage-accounts/list-account-sas

please note that the linting tools do not like the url ListAccountSas because it expects the OperationId to match ListAccountSas -> which then fails validation on using the noun Account in the Verb portion of the operation id.